templates/solutions/data-and-ai/index.html

```diff
@@ -0,0 +1,326 @@
+      </div>
+    {%- endif -%}
+
+  {%- endcall -%}
+
+  <div class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-image-container">
+        {{ image(url="https://assets.ubuntu.com/v1/1444b4e8-data-center.jpg",
+                alt="",
+                width="2464",
+                height="1027",
+                hi_def=True,
+                attrs={"class": "p-image--full-width"},
+                loading="lazy") | safe
+        }}
+      </div>
+    </div>
+  </div>
+
+  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
+    {%- if slot == 'title' -%}
+      <h2>Put your data to work</h2>
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <p>With a robust foundation in place, you can begin unlocking the true value of your data with AI/ML use cases.</p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">MLOps</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        MLOps is the key to bringing AI projects to production at scale. Canonical delivers an open source MLOps platform based on Charmed Kubeflow, designed to automate your machine learning workflows and seamlessly integrate with a wide array of leading ML tools.
+      </p>
+      <div class="p-cta-block">
+        <a href="https://ubuntu.com/ai/mlops">Discover Canonical’s composable approach to MLOps&nbsp;&rsaquo;</a>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">Observability</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        Stay on top of your application performance with open source monitoring tools and dashboards. We provide security and support for a comprehensive collection of best-in-class observability solutions including Prometheus, Grafana, OpenTelemetry, and more.
+      </p>
+      <div class="p-cta-block">
+        <a href="/observability">Explore open source observability&nbsp;&rsaquo;</a>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">RAG</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        Retrieval augmented generation (RAG) is the leading method for enhancing the accuracy and relevance of LLM outputs – and you can easily use Canonical’s Charmed OpenSearch as your open source vector database to enable RAG capabilities.
+        <div class="p-cta-block">
+          <a href="/solutions/ai/genai">Learn more about GenAI infrastructure and RAG&nbsp;&rsaquo;</a>
+        </div>
+      </p>
+    {%- endif -%}
+
+  {%- endcall -%}
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--muted" />
```

Comment:
```suggestion
        <hr class="p-rule" />
```

Shouldn't this be default rule and not muted?